src/clj/bluegenes_tool_store/auth.clj

```diff
@@ -0,0 +1,51 @@
+(ns bluegenes-tool-store.auth
+  (:require [imcljs.auth :as im-auth]
+            [cheshire.core :as cheshire]
+            [ring.util.http-response :as response]
+            [config.core :refer [env]]
+            [clojure.string :as string]))
+
+(defn check-priv
```

Comment:
This function is heavily commented - which is awesome - but docstrings make it easier to generate automated documentation, e.g. for codox - https://github.com/weavejester/codox. A short sentence describing what the function does would be sufficient.